libs/SmartStore/SmartStore/Classes/SFSmartStoreInspectorViewController.m

```diff
@@ -28,7 +28,6 @@
 
 #import "SFSmartStoreInspectorViewController.h"
```

Comment:
@wmathurin We could do so, the WindowManager has provisions for creating new windows.